zuko-api/app/Http/Resources/V1/PublicationsResource.php

```diff
@@ -19,8 +19,7 @@ public function toArray(Request $request): array
             'id' => $this->id,
             'type' => strtoupper($this->type),
             'title' => $this->title,
-            'cover_image_url' => $this->cover_image_url,
-            'url' => $this->attachment()->first()->url()
```

Comment:
A problem arises when attempting to seed for the first time. It cannot seed because there is a migration that removes this field from the table/model.

Reply:
You can leave the seeder or if you want add the seeder to also seed the attachments but no needed to waste time on the seeders just add here the  validation if there is any item in the attachments get the first url if not fallback to empty string for url